### PR TITLE
Exclude Over/Underflow bins in ROOT file when writing Datacards

### PR DESCRIPTION
**Pull request by Bogdan-Wiederspan**
When writing the Datacards, rates exclude flow, while the corresponding ROOT file does not.
To create a match between both the Over and Underflow bin is set to 0.

### CODE REVIEW

**Comment by mafrahm** [comment]
Not necessary to do it in this PR, but could we also add an option to move to overflow content in the last bin? Could maybe even be done similarly to #416

File: columnflow/inference/cms/datacard.py [hunk 1]
```diff
@@ -360,20 +362,66 @@ def write_shapes(
         # create the output file
         out_file = uproot.recreate(shapes_path)
 
+        # helper to remove underflow and overflow values
```
[the remark this answers is not in the document]
Comment (Bogdan-Wiederspan) [reply columnflow/inference/cms/datacard.py]: done

**Comment by Bogdan-Wiederspan** [comment]
>@mafrahm Not necessary to do it in this PR, but could we also add an option to move to overflow content in the last bin? Could maybe even be done similarly to https://github.com/columnflow/columnflow/pull/416 

The PR was extended. When you add a category you have now the option to set a `flow_strategy`. 
Setting this to: `move` is the behavior you want. Options to ignore, remove and warn are also added.

**Review by riga — APPROVED** [review]
LGTM!

**Pull request merged**